site/content/docs/howto/configure-supervisor-with-gitlab.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Configure the Pinniped Supervisor to use Gitlab as an OIDC Provider
+description: Set up the Pinniped Supervisor to use Gitlab login.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Configure Supervisor With Gitlab
```

Comment:
we now have two very similar named "How-to Guides":

![image](https://user-images.githubusercontent.com/3081590/116460057-9797b600-a834-11eb-957c-879d2758c17a.png)

i like that this doc follows a similar pattern to the docs about the `Concierge` (e.g., `Configure Supervisor With Gitlab` =~ `Configure Concierge Webhook Authentication`). have we thought about renaming the `Configure Supervisor` How-to Guide to more specifically talk about setting it up as a downstream issuer? something like `Configure Supervisor OIDC Issuer`? `Configure Supervisor as OIDC Issuer`?